documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -708,6 +714,7 @@ Let's look at what a _create_ event value might look like for our `customers` ta
             "ts_ms": 1520085154000,
             "txId": "6.28.807",
             "scn": 2122185,
+            "commit_scn": 2122185,
```

Comment:
Worth a mention in the section below (where the contents of `source` is described) that this is only present for LogMiner?